galvasr2/deduplicate.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from galvasr2.align.spark.align_lib import load_audio_id_text_id_mapping, load_transcripts
+from datasketch import MinHash, MinHashLSH, MinHashLSHForest
```

Comment:
can you add the appropriate packages to environment.yml? I don't think we have datasketch or nltk right now.